tests/CocktailParty.tex

```diff
@@ -65,21 +64,25 @@ \subsection{Additional rules and remarks}
 
 	\item \textbf{Changing places:} After giving the order (when the robot is not in the party room), the referees may re-arrange the people including their body posture. That is, a sitting person may change to a standing posture and vice versa.
 
-	\item \textbf{Positions and orientations:} All people roughly stay where they are (if not asked to move by the referees), but they are allowed to move in certain limits (e.g. turn around, make a step aside). They do not need to look at the robot, but are requested to do so, when instructed by the robot.
+	\item \textbf{Positions and orientations:} All test participants roughly stay where they are (if not asked to move by the referees), but they are allowed to move in certain limits (e.g. turn around, make a step aside). They do not need to look at the robot, but are requested to do so, when instructed by the robot.
 
-	\item \textbf{Empty arena:} During the test, only the robot, the guest, and the Barman are in the arena. The door opener, the referees and other personnel that is not assigned as test people will be outside the scenario.
+	\item \textbf{Empty arena:} During the test, only the robot, the guest, and the Bartender are in the arena. The door opener, the referees and other personnel that is not assigned as test people will be outside the scenario.
 
-	\item \textbf{Calling instruction:} The team needs to specify before the test which ways of getting the attention of the robot are allowed. This can be waving, calling or both of them. The robot can also decide to skip this part, by asking for people to get close to it.
+	\item \textbf{Calling instruction:} The team needs to specify before the test which ways of getting the attention of the robot are allowed for standing persons. This can be waving, calling or both of them. The robot can also decide to skip this part, by asking for people to get close to it.
+	
+	\item \textbf{Sitting persons:} Sitting persons might have an order but are not actively calling the robot.
 \end{enumerate}
 
 \subsection{Referee instructions}
 
 The referees need to
 \begin{itemize}
-	\item select 5 people and their names from the list of person names (see Section 3.2.8),
-	\item arrange (and re-arrange) people in the party room,
-	\item select the person (barman) who will serve the drinks,
-	\item select the ordering 3 people and the orders to give,
+	\item select at least 5 volunteers and assign names from the list of person names (see \refsec{rule:scenario_names})
+	\item arrange (and re-arrange) people in the party room. At least one is sitting
+	\item assign orders to two standing persons
+	\item assign an order to a sitting person
+	\item select the person (bartender) who will serve the drinks,
+	\item place drinks at the bar while one drink is missing
 	\item in case the robot skips the calling detection, select the ordering person to approach the robot,
```

Comment:
select the ordering person to approach the robot when the robot skips the calling detection